### DISCUSSION TITLE
Google colab Mojo 🔥

### DISCUSSION THREAD

**Discussion by Eprahim-taha** [modular/modular#1596] [General]
How do I make Mojo work inside Google colab 🤔❓

**Comment by mojo-for-ai** [comment]
https://github.com/modularml/mojo/issues/45#issuecomment-1583138120

**Comment by mojo-for-ai** [comment]
Mojo will be available on Google Colab and Replit.com when it becomes open source

**Reply by Eprahim-taha** [reply]
What is the relationship between Mojo being open source and its working on replit and Google colab 🤔
